dev/com.ibm.ws.app.manager.wab.installer_fat/fat/src/com/ibm/ws/app/manager/wab/installer/fat/ConfigurableWABTests.java

```diff
@@ -0,0 +1,208 @@
+/*
+ * IBM Confidential
```

Comment:
@sebratton please remove all IBM Confidential headers, templates and code ... ASAP!!

Reply:
Done. Thanks.